oterm/app/widgets/chat.py

```diff
@@ -117,7 +120,8 @@ async def response_task() -> None:
             message_container.mount(response_chat_item)
             loading = LoadingIndicator()
             message_container.mount(loading)
-            message_container.scroll_end()
+            if autoscroll.parent.can_view(autoscroll):
```

Comment:
Instead of `autoscroll.parent` please use `message_container`. This is cleaner but also satisfied the typing checker.